src/webots/nodes/utils/WbUrl.cpp

```diff
@@ -122,8 +122,10 @@ QString WbUrl::computePath(const WbNode *node, const QString &field, const QStri
 
     if (isLocalUrl(url)) {
       QString newUrl(url);
-      newUrl.replace("webots://", "https://raw.githubusercontent.com/" + WbApplicationInfo::repo() + "/" +
-                                    WbApplicationInfo::branch() + "/");
+      WbVersion version = WbApplicationInfo::version();
```

Comment:
```suggestion
      WbVersion version = WbApplicationInfo::version();
```

Would it be possible to make a const variable here ?

Reply:
Yes, both can actually